docs/news/index.qmd

```diff
@@ -9,6 +9,127 @@ format:
 
 This page provides the release notes associated with each release of RStudio and Posit Workbench. Please contact customer support (<a href="mailto:support@posit.co">support@posit.co</a>) for questions about the described changes.
 
+## RStudio 2024.09.0
+
+**"Cranberry Hibiscus"**
+
+>Date: 2024-09-23
+
+### New
+
+- Posit Product Documentation theme v4.0.2; adds dark mode and several accessibility improvements (rstudio-pro#6373)
+- Added ability for RStudio and Workbench to use AES 256 encryption (rstudio-pro#4011)
+
+#### RStudio
+
+- You can now toggle between R and Python in the Console by clicking on the language's logo in the Console pane header. (#11613)
+- Allow customization of the project scratch path location. (#14875)
+- Restart commands are now run after restoring the search path + global environment by default. (#14636)
+- The "Save as Image" dialog now includes a checkbox "Use device pixel ratio", controlling whether plots are scaled according to the current display's DPI. (#14727)
+- The "Soft-wrap R source files" preference now applies to all source files, and has been re-labelled appropriately. (#10940)
+- RStudio now supports Electron flags set via `~/.config/rstudio/electron-flags.conf` or `~/.config/electron-flags.conf` on Linux / macOS. On Windows, the paths are `%LocalAppData%\RStudio\electron-flags.conf` and `%LocalAppData%\electron-flags.conf`. (#14641)
+- RStudio now uses a glimpse-like display for data.frame list-column entries. (#14257)
+
+#### Posit Workbench
+
+- Quitting Jupyter and VS Code sessions from the home page no longer force quits them immediately; instead, they are given time to exit gracefully, as with RStudio Pro sessions (rstudio-pro#5317)
+- Workbench jobs launched on Kubernetes or on Slurm with Singularity support will now default to using the same container image as the parent session (rstudio-pro#5875)
+- Workbench's Prometheus metrics now track R and Python jobs launched, in addition to sessions (rstudio-pro#6271)
+- When launching sessions on Slurm, environment variables set by the Slurm scheduler itself should now be available to sessions (rstudio-pro#5148, rstudio-pro#4692, rstudio-pro#3255)
+- The `launcher-sessions-forward-container-environment` flag has been deprecated and will be removed in a future version. Use `launcher-sessions-forward-environment=0` to disable environment variable forwarding instead (rstudio-pro#5895)
+- The New Project dialog in RStudio Pro now defaults to the R version of the current session, rather than the system default (rstudio-pro#4244)
+- Workbench jobs launched from RStudio Pro now default to the R version of the current session, rather than the system default (rstudio-pro#5903)
+- Added support for Jupyter Notebook 7 (rstudio-pro#6266)
+- Replaced code server binary with PWB Code Server. PWB Code Server is bundled with the PWB VS Code Extension, so a separate install is no longer required. It is a fork of VS Code 1.89.1. (rstudio-pro#6265)
```

Comment:
Uh oh. We're using a fork of VS Code 1.93 now.

Reply:
Good catch!!

Reply:
```suggestion
- Replaced code server binary with PWB Code Server. PWB Code Server is bundled with the PWB VS Code Extension, so a separate install is no longer required. It is a fork of VS Code. (rstudio-pro#6265)
```